BugTester.java

```diff
@@ -17,6 +17,7 @@ public static void main(String [] args)
       StartDirection = scan.nextInt();
       Bug ladyBug = new Bug(StartPos, StartDirection);
       // We are going to verify if the input that we passed into ladyBug works correctly
+      //Write the code here
```

Comment:
You have to actually write the code here. Once you have written the code we can pull.